src/MICore/Debugger.cs

```diff
@@ -544,15 +568,10 @@ public Task CmdBreakInternal()
         {
             this.VerifyNotDebuggingCoreDump();
 
-            // TODO May need to fix attach on windows.
-            // Note that interrupt doesn't work when attached on OS X with gdb:
+            // Note that interrupt doesn't work on OS X with gdb:
```

Comment:
Can you reset line 571 to what it was before (specifying it's only when attached)